pkgs/by-name/os/osu-lazer-bin/package.nix

```diff
@@ -95,7 +95,7 @@ else
           --set OSU_EXTERNAL_UPDATE_PROVIDER 1
         install -m 444 -D ${contents}/osu!.desktop -t $out/share/applications
         for i in 16 32 48 64 96 128 256 512 1024; do
-          install -D ${contents}/osu!.png $out/share/icons/hicolor/''${i}x$i/apps/osu!.png
+          install -D ${contents}/osu.png $out/share/icons/hicolor/''${i}x$i/apps/osu.png
```

Comment:
(partial) diff between the previous and this AppImage:

```diff
diff -r old/osu!.desktop new/osu!.desktop
2d1
< 
8,9c7,8
< Icon=osu!
< Exec=osu!
---
> Icon=osu
> Exec=osu! %u
15c14
```

I assume we should do the same for the `osu-lazer` too